### DISCUSSION TITLE
Is there a limit on number of edges

### DISCUSSION THREAD

**Discussion by candcconsulting** [reaviz/reagraph#196] [Q&A]
I have a fairly complex graph
385 Classes and 1799 Relationships
Here you can see IFCDynamic.IFCFurnitureElement has no edges
![image](https://github.com/reaviz/reagraph/assets/19268085/d72a1f4e-1e96-496e-93ca-f2b2526534f7)
However in the relationships
IFCFurniture has many relationships
![image](https://github.com/reaviz/reagraph/assets/19268085/a4bb44df-a5e6-4bd6-b839-3a4f557b10cb)
I do not see any warnings, other than a dangling class

**Comment by amcdnl** [comment]
It should work - can you submit a code example?

**Reply by candcconsulting** [reply]
Thanks @amcdnl I have checked this and yes it looks like the data there is some "sameas" information which we are not handling correctly.  Checking further, but it looks like the dangling nodes all have equivalents.
